internal/cmd/connect/command_plugin.go

```diff
@@ -6,23 +6,31 @@ import (
 	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
 
 	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
 )
 
 type pluginCommand struct {
 	*pcmd.AuthenticatedCLICommand
 }
 
-func newPluginCommand(prerunner pcmd.PreRunner) *cobra.Command {
+func newPluginCommand(cfg *v1.Config, prerunner pcmd.PreRunner) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:         "plugin",
-		Short:       "Show plugins and their configurations.",
-		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+		Use:   "plugin",
+		Short: "Manage Connect plugins.",
 	}
 
-	c := &pluginCommand{pcmd.NewAuthenticatedCLICommand(cmd, prerunner)}
+	c := &pluginCommand{}
 
-	cmd.AddCommand(c.newDescribeCommand())
-	cmd.AddCommand(c.newListCommand())
+	if cfg.IsCloudLogin() {
+		c.AuthenticatedCLICommand = pcmd.NewAuthenticatedCLICommand(cmd, prerunner)
+
+		cmd.AddCommand(c.newDescribeCommand())
```

Comment:
We want users to get a message along the lines of "Please log in to Confluent Cloud" if they're logged in to Confluent Platform but they run `confluent connect plugin describe`. Can you move this line outside of the `if`, and make sure it has the correct annotation? I think this is what we want:
```
Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin}
```